src/main/java/org/carlspring/strongbox/janusgraph/graph/schema/StrongboxSchema.java

```diff
@@ -23,9 +24,10 @@
     private static final Logger logger = LoggerFactory.getLogger(StrongboxSchema.class);
 
     @Inject
-    public void createSchema(JanusGraphFactory.Builder jgf)
+    public void createSchema(JanusGraph jg)
```

Comment:
@fuss86 can you please leave it like it was originally? (the reason is in above comments)

Reply:
ignore